README.md

```diff
@@ -1,2 +1,171 @@
+- Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add_tasks]**
+- **[Remove_tasks]**
+- **[ave_in_local_storage]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- Use `git clone git@github.com:mrkamin/Leaderboard.git` to clone the repository using the HTTPS link into your local machine
+
+### Setup
+
+- `cd` into your local repository
+
+### Install
+
+- `npm install` - Installs the necessary dependencies
+
+### Usage
+
+- Open the index file and enjoy
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Mohammad Rafi Amin**
+
+- GitHub: [@githubhandle](https://github.com/mrkamin)
+- Twitter: [@twitterhandle](https://twitter.com/Mohamma63974237)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohammad-rafi-amin-63b4319b/)
+
+👤 **Author 2**
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Add unique ID for each tasks]**
+- [ ] **[Add tasks to the taks list]**
+- [ ] **[edit tasks in the tasks list]**
+- [ ] **[delete tasks in the tasks list]**
+- [ ] **[delete all tasks in the tasks list]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page]().
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project please support us by giving it a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank  in Behance for the original design idea: []().
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[What frameworks have you used?]**
+
+  - [We have not used any frameworks]
+
+- **[Why did you work in group for a simple project?]**
+
+  - [Working in group is a habit of real work places and is good practice.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job documenting your code! 🚀 However, I strongly suggest you fix this LICENSE link by adding an MIT license to your project! 🤓 🛠️